taxcalc/tests/test_policy.py

```diff
@@ -988,9 +907,9 @@ def test_index_offset_reform():
     pvalue2 = dict()
     for cyr in [2019, 2020, 2021]:
         policy1.set_year(cyr)
-        pvalue1[cyr] = policy1.CTC_c
+        pvalue1[cyr] = policy1.CTC_c[0]
         policy2.set_year(cyr)
-        pvalue2[cyr] = policy2.CTC_c
+        pvalue2[cyr] = policy2.CTC_c[0]
```

Comment:
This just brings the parameter down a dimension. This is related to the API change I mentioned in the initial comment of this PR.